compile.c

```diff
@@ -9504,15 +9508,15 @@ compile_super(rb_iseq_t *iseq, LINK_ANCHOR *const ret, const NODE *const node, i
             /* post arguments */
             int post_len = local_body->param.post_num;
             int post_start = local_body->param.post_start;
+            flag |= VM_CALL_ARGS_SPLAT_MUT;
```

Comment:
Is it OK even if `!has_rest`?

Reply:
No, I pushed a commit to move this into the has_rest block.